ordinals/hooks/pre-start

```diff
@@ -24,6 +24,11 @@ if [[ -f "${APP_DATA_DIR}/index.redb" ]]; then
   set_correct_permissions "${APP_DATA_DIR}"
 fi
 
+# re-index the ordinals data for runes
+if [[ ! -f "${APP_DATA_DIR}/INDEXED_0-17-1" ]]; then
```

Comment:
Nice idea here

Reply:
Thanks @mrv777,  it's a little hacky but will work for now. I've renamed the file to `AT_LEAST_0-17-1` since future installs will have this written to file.

Reply:
Another idea I had but haven't had time today to think it through for pros/cons would be a VERSION file that just has the version number as text in it.  The script can then just check the contents of the file to see what it should do, and update it at the end.

I know quite a few people excited for this to be updated though as they prefer the official umbrel apps and your other fix to simplify the ord commands will be greatly appreciated.

Great work and thank you!

Reply:
That's a good idea if this starts getting out of hand because it will keep the data directory cleaner and less confusing for someone who views it and sees a bunch of strangely named files